src/components/commons/CardList.tsx

```diff
@@ -1,5 +1,28 @@
 import styled from "styled-components";
-import Card from "./Card";
+import Card from "./Card.tsx";
+
+interface selectFolderLinksProps {
```

Comment:
인터페이스 선언은 파스칼 케이스로 하는 게 좋습니다.
```suggestion
interface SelectFolderLinksProps {
```

(멘토링 중 잠시 들렸습니다 😃 )